content/docs/dvclive/api-reference/set_step.md

```diff
@@ -0,0 +1,42 @@
+# dvclive.set_step()
+
+Updates the `step` value.
+
+```py
+def set_step(step: int):
+```
+
+#### Usage:
+
+```py
+import dvclive
+
+for step in [0, 10, 20]:
+    dvclive.set_step(step)
+    dvclive.log("metric_1", 0.9)
+    dvclive.log("metric_2", 0.7)
+    dvclive.next_step()
```

Comment:
I thought it's not necessary to have both `set_step` and `next_step`?

Reply:
Correct